src/Capgemini.PowerApps.PackageDeployerTemplate/PackageTemplateBase.cs

```diff
@@ -10,13 +11,14 @@ namespace Capgemini.PowerApps.PackageDeployerTemplate
 {
     public abstract class PackageTemplateBase : ImportExtension
     {
-
         [System.Diagnostics.CodeAnalysis.SuppressMessage(
             "Major Code Smell",
             "S1144:Unused private types or members should be removed", 
             Justification = "Required or Polly does not get copied when referenced via project reference (e.g. in the TestPackage project)")]
         private readonly Polly.Policy _policy;
 
+        public List<string> _solutions { get; private set; }
```

Comment:
I think this one might come up as an analysis issue on styling - I believe no leading underscores and pascal-casing for public/protected members is the default

Reply:
You are right Max it may moan.